[FIX] price_security: Repeated sequence field in pricelist tree view

zaoral opened the pull request.

zaoral committed:
[FIX] price_security: Delete sequnce from pricelist list view

This one appears twice in the view.
Remove the view since is not needed in version 11.0

zaoral committed:
[FIX] price_security: Remove execution permission from view files

zaoral committed:
[IMP] price_security: separete view files per model

nicomacr reviewed (dismissed):
@zaoral thanks for the PR.
 Tecnical review +1

nicomacr commented on price_security/views/product_pricelist_views.xml:
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```
@zaoral this it's not necessary. This show the field in any place. Like see in this image:
![seleccion_454](https://user-images.githubusercontent.com/7657373/41035949-b463f742-6964-11e8-95aa-587ed2247ae5.png)

zaoral committed:
[FIX] price_security: remove product pricelist view

This one was adding the sequence to the pricelist form view. This is not
neccesary and it was returning an error in the view.

nicomacr reviewed (approved):
great @zaoral
